noisyatom/aboutus/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```

Comment:
Why did you create a test.py file with nothing in it?

Reply:
When you create a new app in Django it adds a tests.py file. This is a reminder that we have to add test cases.

Reply:
> There is still the app 'products' in the INSTALLED APPS inside settings.py. This should have been removed. It is updated now and I updated the settings.py file.
> 
> Please test if you are able to add members to the team.

I agree to remove the products in setting.py file because we don't have apps for products yet. Last night I tried to find the bug have not checked if there is products apps, so I agree with Christine